packages/dx-react-grid/docs/guides/sorting.md

```diff
@@ -41,6 +41,12 @@ Note that the `LocalGrouping` plugin should follow the `LocalSorting` to provide
 
 .embedded-demo(sorting/local-group-sorting)
 
+## Custom Sorting Algorithm
+
+The [LocalSorting](../reference/local-sorting.md) plugin's `getColumnCompare` property allows you to implement a custom sorting algorithm. If the `getColumnCompare` function returns undefined, the default sorting algorithm is applied.
+
+.embedded-demo(sorting/local-custom-sorting)
+
 ## Remote Sorting
 
 You can handle the Grid sorting state changes to request data from the server with the corresponding sorting applied if your data service supports sorting operations.
```

Comment:
You can handle the Grid sorting state changes to request data from the server with the corresponding sorting applied if your data service supports sorting operations." This sentence doesn't make sense. Please clarify.